docs/bakers.md

```diff
@@ -232,40 +241,48 @@ tezos:
       block:              # Standard block baking
       attestation:        # Standard attestations
       preattestation:     # Pre-attestations  
-      attestation_with_dal: # ✨ Required for DAL attestations
+      attestation_with_dal: # Required for DAL attestations
       generic:
         - transaction
         - reveal
         - delegation
         - stake
 ```
 
-:::note Terminology
-Signatory supports both old (`endorsement`/`preendorsement`) and modern (`attestation`/`preattestation`) terminology in policy configuration. Both work identically, but we recommend using the modern terms to match current Tezos protocol terminology.
+:::warning Important
+Without `attestation_with_dal` in your Signatory policy, DAL attestation requests will be rejected, and you'll miss those rewards.
 :::
 
-**DAL Setup Overview**
+### Key Requirements
 
-To participate in DAL attestations:
+**BLS Consensus Keys (tz4)**: A **companion key is mandatory** for DAL attestations. Without it, tz4 bakers cannot produce DAL attestations.
 
-1. **Run a DAL node** alongside your Tezos node
-2. **Configure your baker** to use the DAL node (`--dal-node` flag)
-3. **Update Signatory policy** to allow `attestation_with_dal` operations
-
-:::warning Important
-Without `attestation_with_dal` in your Signatory policy, DAL attestation requests will be rejected, and you'll miss those rewards.
-:::
+```bash
+# Register companion key for existing delegate
+octez-client set companion key for <manager_key> to <companion_key>
+```
 
-**Consensus Keys and DAL**
+**DAL Operations**:
+- **`attestation_with_dal`**: Required for DAL participation
+  - Standard keys (tz1, tz2, tz3): Single consensus key signature
+  - BLS keys (tz4): Requires both consensus key AND companion key signatures
+- **`dal_entrapment_evidence`**: **DO NOT ALLOW** - Anonymous operation, no signature needed
+- **`dal_publish_commitment`**: **DO NOT ALLOW** - Only needed if publishing data to DAL
 
-While consensus keys are not strictly required for DAL operations, **using consensus keys is considered current best practice** for production baking setups. Consensus keys provide operational flexibility and security benefits by separating the baker's identity (manager key) from the key used for consensus operations (baking, attesting).
+### Setup Overview
 
-**Further Reading**
+1. **Run a DAL node** alongside your Tezos node (see [DAL Node Setup Guide](https://octez.tezos.com/docs/shell/dal_run.html))
+2. **Configure your baker** to use the DAL node (`--dal-node` flag)
+3. **Update Signatory policy** to allow `attestation_with_dal` operations
 
-The details of running DAL nodes and collecting DAL attestation rewards are outside the scope of this Signatory guide. For comprehensive information:
+```bash
+# Start baker with DAL integration
```

Comment:
This command will not work to get DAL attestations. You have to path the consensus and companion keys.  for example, ```octez-baker  run with local node ~/.tezos-node \
    --liquidity-baking-toggle-vote pass \
    --dal-node http://localhost:10732 \
    consensus_key companion_key```